Zend/zend_alloc.c

```diff
@@ -462,9 +462,16 @@ static void *zend_mm_mmap(size_t size)
 #else
 	void *ptr;
 
-#ifdef MAP_HUGETLB
+#if defined(MAP_HUGETLB) || defined(VM_FLAGS_SUPERPAGE_SIZE_2MB)
```

Comment:
will this also cover AppleSilicon macos?

Reply:
if you mean mac m1 I did the dev on one :)